src/v/cluster/config_manager.cc

```diff
@@ -658,6 +659,8 @@ apply_local(const cluster_config_delta_cmd_data& data, bool silent) {
 
             auto validation_err = property.validate(val);
             if (validation_err.has_value()) {
+                auto msg = features::enterprise_error_message::cluster_property(
+                  u.key, val);
```

Comment:
this looks new. I'm surprised we print an enterprise error on a straight validation error. am I missing something?